src/Worker/Core/Logs.cs

```diff
@@ -15,5 +16,8 @@ static partial class Logs
     {
         [LoggerMessage(EventId = 55, Level = LogLevel.Information, Message = "{instanceId}: Evaluating custom retry handler for failed '{name}' task. Attempt = {attempt}.")]
         public static partial void RetryingTask(this ILogger logger, string instanceId, string name, int attempt);
+
+        [LoggerMessage(EventId = 80, Level = LogLevel.Error, Message = "Unhandled exception in entity operation {EntityInstanceId}/{OperationName}.")]
+        public static partial void OperationError(this ILogger logger, Exception ex, EntityInstanceId entityInstanceId, string operationName);
```

Comment:
wow, we jumped a fair bit: from 55 to 80. Why?

Reply:
Our event IDs are all over the place - I was trying to avoid the IDs in DurableTask.Core

Reply:
Went with an ID of 15 as it is in line with the other eventIds we have (from the gRPC worker)